public/stylesheets/site/2.0/13-group-blurb.css

```diff
@@ -369,17 +369,18 @@ blurbs on the manage collection items pages, mostly reseting styles inherited fr
 	min-height: 30px;
 }
 
-.bookmark .user .heading {
+.bookmark .user .byline.heading {
```

Comment:
For the sake of simplicity (quite necessary with as many CSS files as we have! 😆) we generally try to avoid chaining class names. Is there a specific page where there's a problem if we just use `.bookmark .user .byline`?

Reply:
Haha makes sense! It's been a while since I've worked in pure CSS and I forget how careful you've gotta be with the selectors sometimes. I looked around and it seems like just `.bookmark .user .byline` works completely fine, so I just pushed an update to make it that. Thanks!